tools/mkdocs_macros_bids/__init__.py

```diff
@@ -5,6 +5,7 @@
     make_filetree_example,
     make_metadata_table,
     make_suffix_table,
+    render_text,
```

Comment:
is there a reason why only some of the functions from macros.py are listed here?

Reply:
I have no idea what the story is with this package, and have left it as-is because it works. @tsalo probably knows more.

Reply:
ha ok

Reply:
I think it was just there to make importing easier. Any missing ones are probably just the result of not keeping the init up to date. I don't think anything actually uses `mkdocs_macros_bids.X` instead of `mkdocs_macros_bids.macros.X` though.

Reply:
OK should maybe clear up this init file a bit then?

Reply:
Yeah that should be fine.

Reply:
@tsalo 
should I do this in this PR? Asking because currently I think one is ready to go.